andela/arakari-rc
# #139284023 Embed social content

#### What does this PR do?
Embeds social content on the app.

 #### Description of Task to be completed?
Modify views to add social content (twitter and Facebook) to the RC pages. When viewing a product a user is able to share the product on their social space. When viewing a successful order, a user is able to share the confirmation order. Finally when on the shop's index page, AFTER the products list, a vendor is able to add social timelines for the shop's social timelines allowing users to interact with the timelines.

 #### How should this be manually tested?
- log in as Admin
- Enable social channel configuration and provide configuration options
- Visit Index page, order summary page to see changes

 #### Any background context you want to provide?


 #### What are the relevant pivotal tracker stories?
#139284023
 #### Screenshots (if appropriate)
![image](https://cloud.githubusercontent.com/assets/24469235/23692489/24dda0a4-03e0-11e7-8ec2-ddd0cddffec6.png)

![image](https://cloud.githubusercontent.com/assets/24469235/23692494/2e7c1c1c-03e0-11e7-99fa-467d14967814.png)

 #### Questions:
